tests/Unit/NumericalAlgorithms/Spectral/Test_Filtering.cpp

```diff
@@ -0,0 +1,83 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <cmath>
+#include <cstddef>
+#include <vector>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Matrix.hpp"
+#include "DataStructures/ModalVector.hpp"
+#include "Domain/Mesh.hpp"
+#include "NumericalAlgorithms/LinearOperators/CoefficientTransforms.hpp"
+#include "NumericalAlgorithms/Spectral/Filtering.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+#include "Utilities/Blas.hpp"
+
+namespace {
+
+template <Spectral::Basis BasisType, Spectral::Quadrature QuadratureType>
+void test_exponential_filter(const size_t min_pts, const double alpha,
+                             const unsigned half_power,
+                             const double eps) noexcept {
+  Approx local_approx = Approx::custom().epsilon(eps).scale(1.0);
+  CAPTURE(BasisType);
+  CAPTURE(QuadratureType);
+  CAPTURE(eps);
+  for (size_t num_pts = min_pts;
```

Comment:
Here as well.
Instead of passing `min_pts` just use `Spectral::minimum_number_of_points<BasisType,QuadratureType>`

Reply:
:+1: done!